src/components/SearchModal/ImportList.tsx

```diff
@@ -17,7 +17,7 @@ import { TokenList } from '@uniswap/token-lists'
 import { useDispatch } from 'react-redux'
 import { AppDispatch } from 'state'
 import { useFetchListCallback } from 'hooks/useFetchListCallback'
-import { removeList, enableList } from 'state/lists/actions'
+import { removeList, enableList } from '@src/state/lists/actions'
```

Comment:
why do you use '@src/state/lists/actions' ?

Once you override a module, like in this case, I would use our custom

Reply:
in this case this is an escape hatch

Reply:
i need them to use the defaults

Reply:
aha, ok, then it make sense